deploy/crds/catalogsourceconfig.crd.yaml

```diff
@@ -29,6 +29,14 @@ spec:
     type: string
     description: Current status of the CatalogSourceConfig
     JSONPath: .status.currentPhase.phase.name
+  - name: DisplayName
+    type: string
+    description: Display Name for resulting CatalogSource
+    JSONPath: .spec.displayName
+  - name: Publisher
+    type: string
+    description: Publisher for source propagated to CatalogSource
+    JSONPath: .spec.publisher
```

Comment:
Should be prefix it with `cs` i.e. `.spec.csPublisher` to call out it is for the `CatalogSource`?